Neos.Flow/Classes/Security/AccountInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Neos\Flow\Security;
+
+use Neos\Flow\Security\Authentication\AuthenticationProviderName;
+use Neos\Flow\Security\Authentication\CredentialsSource;
+
+interface AccountInterface {
+
+	/**
+	 * @return AccountIdentifier
+	 */
+	public function getIdentifier() : AccountIdentifier;
+
+	/**
+	 * @return AuthenticationProviderName
+	 */
+	public function getProviderName() : AuthenticationProviderName;
+
+	/**
+	 * @return mixed
+	 */
+	public function getCredentialsSource() : CredentialsSource;
+}
```

Comment:
Codestyle demands a newline at the end of files

Reply:
This is a PSR-2 thing? So I must change to spaces for indention I guess :)